jwst/fringe/fringe.py

```diff
@@ -65,8 +61,13 @@ def apply_fringe(input_model, fringe):
     # to pixels having good calibration values
     fringe_data = fringe.data
     fringe_data[np.isnan(fringe_data)] = 1.0
+    inv_fringe_data_sq = 1 / (fringe_data * fringe_data)
```

Comment:
Multiplication is slightly faster than division. Since this will be applied 3x, I pre-calculated its inverse squared. Might be pre-mature optimization but doesn't hurt?